Microsoft.DotNet.Interactive.PowerShell/PowerShellKernel.cs

```diff
@@ -148,7 +138,7 @@ public override bool TryGetVariable(string name, out object value)
                 return Task.CompletedTask;
             }
 
-            if (cancellationSource.IsCancellationRequested)
+            if (context.CancellationToken.IsCancellationRequested)
```

Comment:
Does this get canceled when the user hits stop in the Jupyter GUI?

If so, I'd like to see this cancel the PowerShell execution if possible.

Reply:
@jonsequitur made the change to lift up the support to cancel an operation, but it's not entirely wired up yet (take a look at that property in KernelInvocationContext). I will submit a separate PR to cancel PowerShell execution when it's wired up.